subsys/net/ip/Kconfig

```diff
@@ -140,9 +140,9 @@ config NET_SHELL_DYN_CMD_COMPLETION
 
 config NET_TC_TX_COUNT
 	int "How many Tx traffic classes to have for each network device"
-	default 1
-	range 1 NET_TC_NUM_PRIORITIES if NET_TC_NUM_PRIORITIES<=8
-	range 1 8
+	default 0
```

Comment:
Are we really ready to change the default like that? How will we test config with threads?